### DISCUSSION TITLE
dynamic audio visualizer

### DISCUSSION THREAD

**Discussion by seantai** [katspaugh/wavesurfer.js#3345] [Q&A] [answered]
Trying to add a dynamic audio visualizer to a project that already uses wavesurfer 7.0
Because the project plays audio via wavesurfer, I believe I need to use the same AudioContext that wavesurfer uses, in order to create the correct analyser. 
The current web audio example creates a new AudioContext, which does not solve this context. 

Possible to make this prop public? Open to any solutions.

**Comment by katspaugh** [comment, marked as answer]
You don't need to use the same audiocontext. Just create your own.
E.g. https://wavesurfer.xyz/examples/?791a4255945c9abaee2674976c7c034a

**Reply by seantai** [reply]
this works nicely. thank you for the quick reply!
